packages/clay-navigation/src/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import classNames from 'classnames';
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+export interface IBreadcrumbItem {
+	/**
+	 * Flag to indicate if the Breadcrumb item is active or not.
+	 */
+	active?: boolean;
+
+	/**
+	 * This value is used to be the target of the link.
+	 */
+	href?: string;
+
+	/**
+	 * Label of the Breadcrumb item
+	 */
+	label: string;
+
+	/**
+	 * Callback for when a Breadcrumb item is clicked.
+	 */
+	onItemClick?: (event: React.SyntheticEvent) => void;
+}
+
+export const BreadcrumbItem = ({
+	active,
+	href,
+	label,
+	onItemClick,
+	...otherProps
+}: IBreadcrumbItem) => {
+	const ItemElement = ({
+		active,
+		href,
+		label,
+		onItemClick,
+	}: IBreadcrumbItem) => {
+		if (href && !active) {
+			return (
+				<a
+					className="breadcrumb-link"
+					data-testid={`testId${label}`}
+					href={href}
+					role="button"
+				>
+					{label}
+				</a>
+			);
+		}
+
+		if (onItemClick) {
```

Comment:
I think we can do the same here that was done in `href` just render the `ClayButton` if the item is not active.